tests/aws/services/events/test_events_targets.snapshot.json

```diff
@@ -669,5 +669,104 @@
         }
       ]
     }
+  },
+  "tests/aws/services/events/test_events_targets.py::TestEventsTargetApiGateway::test_put_events_with_target_api_gateway": {
+    "recorded-date": "19-09-2024, 19:32:01",
+    "recorded-content": {
+      "create_lambda_response": {
+        "CreateEventSourceMappingResponse": null,
+        "CreateFunctionResponse": {
+          "Architectures": [
+            "x86_64"
+          ],
+          "CodeSha256": "sXbjOPfOag8wOGYl3EzZyMHpQOsmtwP4xiaKKO8DZsM=",
```

Comment:
Thought this would've been handled by `transform_lambda_api()` 🤔 I'd consider adding a transformer for this field to transform just the key_value at `"CodeSha256"` (hint!).